src/client.js

```diff
@@ -118,6 +119,15 @@ class Client {
   }
 
   /**
+   * A wrapper around the constructor, for backwards compatability.
```

Comment:
Even the artifact we build exports the single `ShopifyBuy` global, and `buildClient` is the entry point.